firedrake/adjoint/ensemble_reduced_functional.py

```diff
@@ -38,6 +38,11 @@ class EnsembleReducedFunctional(ReducedFunctional):
     ensemble : Ensemble
         An instance of the :class:`~.ensemble.Ensemble`. It is used to communicate the
         functionals and their derivatives between the ensemble members.
+    scatter_control : bool
+        If ``True``, the functionals and their derivatives are summed over the ensemble
+        communicator ``ensemble.ensemble_comm``. In this case, one assumes one control
+        (or a list of control) is scattered over the ensemble members. If ``False``,
+        ``EnsembleReducedFunctional`` methods will not apply the sum operation.
```

Comment:
This conflates the scattering of the control with the summing of the result. As discussed earlier, `scatter_control` only has to do with the first of these.